src/widget/paintable.cpp

```diff
@@ -159,7 +159,7 @@ void Paintable::draw(const QRectF& targetRect, QPainter* pPainter,
                          math_min(sourceRect.height(), targetRect.height()));
         QRectF adjustedTarget(targetRect.topLeft(), fixedSize);
         QRectF adjustedSource(sourceRect.topLeft(), fixedSize);
-        return drawInternal(adjustedTarget, pPainter, adjustedSource);
+        drawInternal(adjustedTarget, pPainter, adjustedSource);
```

Comment:
We should use a switch/case here like we did in the `BaseTrackTableModel` ;)

Reply:
Done.